### PR TITLE
For TSE Review

### PR DESCRIPTION
**Pull request by paulo-techie**
This Pull Request recreates thenextweb.com homepage.

The focus was on responsiveness and which has been achieved mostly through flexbox.

The layout achieves responsiveness over several devices with different screen sizes.

Project includes one media query to preserve layout integrity for devices with less than 768 pixels in width. These include ipads, tablets and phones.

Code has been validated for errors.

A descriptive readme file is included together with a screenshot of the main page.

Main technologies used are CSS3 and HTML5 almost exclusively.

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -0,0 +1,557 @@
...
+
+
+button {
+    background: #333;
+    border-color: #333;
+    border-radius: 50%;
+    color: #fff;
+    font-size: large;
+    height: 40px;
+    width: 40px;
```
Comment (MauricioRobayo) [inline style.css]: You could remove the default border from the buttons (border: none) and may be set here the line-height: 1.

**Review by MauricioRobayo — CHANGES_REQUESTED** [review]
Hi, this is great work 🌟!

You are nailing it, but need some improvement before approving it:

- [ ] Make sure you are using the [.stickler file from the linters-config repo](https://github.com/microverseinc/linters-config/blob/master/css/.stickler.yml).

Navigation bar - Medium version:
- [ ] TWN logo appears at the leftmost part ([link to page](https://thenextweb.com/))

Top 3 articles section - Medium version:
- [ ] TWN logo disappears ([link to page](https://thenextweb.com/))

On the categories section:
- [ ] Update each section title to use the section name: Apps, Gear, Tech, etc. Currently all of them are using `Apps`.

Original: 

image

Yours:

image

I can see you included `.DS_Store` in the `.gitignore` file, but maybe you did that after you committed that file, so it is showing here. You can use the [`git rm`](https://git-scm.com/docs/git-rm) command to remove it.

Status: **Request changes**.

File: .stickler.yml [hunk 2]
```diff
@@ -0,0 +1,5 @@
+linters:
```
Comment (MauricioRobayo) [inline .stickler.yml]: This file should be the one from the linters-config repo: https://github.com/microverseinc/linters-config/blob/master/css/.stickler.yml

Comment (mcrd25) [inline .stickler.yml]: I noticed you mentioned in a commit that it doesn't work wit hMicroverse recommended config. Are you sure?
See previously mentioned link and this link https://github.com/microverseinc/linters-config/blob/master/css/stylelint.config.js for style config js file.

**Review by mcrd25 — APPROVED** [review]
# Changes Approved 🤓 💪 

Hi, @paulo-techie and @kbjude, 👋 

You did a great job! 👏 
My only concern is that you did not use the stickler config recommended.
Please try to do so before merging with my comment as a guideline. But for now, I'd give it a pass because you have no linting errors present.

### Additional Suggestions [Nice to Haves]
- Try and put your assets into a separate folder (image assets, CSS files, etc. They may even be in their own subfolders based on the type of file)


***If you have any questions, now or in the future, do not hesitate to contact me on social media, email or on slack and I will be glad to chat with you.***

*Happy coding 😄,*
**[--Maya](https://github.com/mcrd25)**
[GitHub](https://github.com/mcrd25) [Twitter](https://twitter.com/mcrd25) [linkedin](https://www.linkedin.com/in/mayadouglas/) [GitLab](https://gitlab.com/mcrd25)

File: .stickler.yml (hunk 2 above)
Comment (paulo-techie) [reply .stickler.yml]: Thanks Maya. I will try it again. Cheers.

Comment (paulo-techie) [reply .stickler.yml]: Yeah. It has worked now. I guess I rushed the first time. Thanks again!

Comment (mcrd25) [reply .stickler.yml:2]: Yay! 🙂

**Pull request merged**